New options for Prometheus rw output (#1366)
grafana/k6-docs: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md

| Name | Type |  Description |
| ---- | ---- |  ----------- |
| `K6_PROMETHEUS_RW_SERVER_URL` | `string`  | URL of the Prometheus remote write implementation's endpoint. Default is ``http://localhost:9090/api/v1/write`` |

> | `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |

Reviewer: ```suggestion
| `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | An additional header to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |
```

Author: Resolved by https://github.com/grafana/k6-docs/pull/1366#discussion_r1350486668

> | `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |

Reviewer 2: I spotted a couple of things here:
1. we refer to the parameter as a list of `string`, but we mention, and give an example, of using only a single one. Maybe we could rephrase to outline what I understand as being is meant: multiple headers of the same format with different `<header-key>` can be set in the environment.
2. In the example the `<header-key>` is prefixed with `_X` but not in the description, was it on purpose? 

```suggestion
| `K6_PROMETHEUS_RW_HEADERS_X-<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. e.g. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |
```

What do you think?

Author: ```suggestion
| `K6_PROMETHEUS_RW_HEADERS_<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. For example, `K6_PROMETHEUS_RW_HEADERS_MY-HEADER-CUSTOM-KEY=foo`. |
```

> | `K6_PROMETHEUS_RW_HEADERS_<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. For example, `K6_PROMETHEUS_RW_HEADERS_MY-HEADER-CUSTOM-KEY=foo`. |

Author: ```suggestion
| `K6_PROMETHEUS_RW_HEADERS_<header-key>` | `string` | An additional header to include in the HTTP requests can be set using this described format. For example, `K6_PROMETHEUS_RW_HEADERS_MY-HEADER-CUSTOM-KEY=foo`. |
```

| `K6_PROMETHEUS_RW_HTTP_HEADERS` | comma-separated list of key values | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HTTP_HEADERS=key1:value1,key2:value2`. |
| `K6_PROMETHEUS_RW_PUSH_INTERVAL` | `string` | Interval of the metrics' aggregation and upload to the endpoint. Default is `5s`. |
| `K6_PROMETHEUS_RW_TREND_AS_NATIVE_HISTOGRAM` | `boolean` | If true, it maps the all defined trend metrics as [Native Histograms](#prometheus-native-histogram). Default is `false`. |
| `K6_PROMETHEUS_RW_TREND_STATS` | list of `string` |  If Native Histogram is not enabled then it defines the stats functions to map for the all defined trend metrics. It's a comma-separated list of stats functions to include (e.g. `p(90),avg,sum`). Check the trend section to know the entire set of the supported stats. Default is `p(99)`. |
| `K6_PROMETHEUS_RW_INSECURE_SKIP_TLS_VERIFY` | `boolean` | If true, the HTTP client skips TLS verification on the endpoint. Default is `false`. |
| `K6_PROMETHEUS_RW_STALE_MARKERS` | `boolean` | If true, the output at the end of the test marks all the seen time series as stale. Default is `false`. |

> | `K6_PROMETHEUS_RW_USERNAME` | `string` | User for the HTTP Basic authentication at the Prometheus remote write endpoint. |

Reviewer: ```suggestion
| `K6_PROMETHEUS_RW_USERNAME` | `string` | Username for the HTTP Basic authentication at the Prometheus remote write endpoint. |
```

Reviewer 2: Interestingly enough the [RFC](https://datatracker.ietf.org/doc/html/rfc7617) refers to the user name as "user-id". But I acknowledge that it's probably used as "username" all over the place regardless 😄

| `K6_PROMETHEUS_RW_PASSWORD` | `string` | Password for the HTTP Basic authentication at the Prometheus remote write endpoint. |

> | `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | Certifivate to authenticate via mTLS. |

Reviewer 2: ```suggestion
| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | Certificate to authenticate via mTLS. |
```

Corrects a typo in the word "certificate". On another note, I think "via" is a very latin-based language wording (we use extensively in french too), but I'm not sure it's that common in English though, @heitortsergent might know more 🤔

> | `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | Certificate to authenticate via mTLS. |
> | `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE_KEY` | `string` | Key to authenticate via mTLS. |

Reviewer: What do you think about being more explicit that this is a path, not the data?
```suggestion
| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | A path to the PEM (Privacy-Enhanced Mail) formatted client certificate. |
| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE_KEY` | `string` | A path to the PEM formatted client private key. |
```

> | `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` |  Sets the Authorization Bearer Token Header. |

Author: ```suggestion
| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` | Sets the Authorization Bearer Token Header. |
```

> | `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` | Set the Authorization header following the Bearer token format. |

Reviewer: Maybe?
```suggestion
| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` | A bearer token for authorization. |
```

Reviewer 2: I would suggest

```suggestion
| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` |  Sets the Authorization Bearer Token Header. |
```
